File: src/main/java/com/p2p/lending/service/InvestService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.InvestInfo;

public interface InvestService {
	public int investA(InvestInfo ii);
	public List<InvestInfo> investS(Map<String, Object> map);
	public Double sumMoney(Map<String, Object> map);
	
	public List<InvestInfo> getDtail(Map<String, InvestInfo> map);
	public Integer getMoney(Integer uid);
}

File: src/main/java/com/p2p/lending/service/InvestService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.InvestInfo
import org.springframework.stereotype.Service

@Service
interface InvestService {
    fun investA(ii: InvestInfo): Int
    fun investS(map: Map<String, Any>): List<InvestInfo>
    fun sumMoney(map: Map<String, Any>): Double?

    fun getDtail(map: Map<String, InvestInfo>): List<InvestInfo>
    fun getMoney(uid: Int?): Int?
}
